doc-surrealdb_versioned_docs/version-latest/introduction/concepts/namespace.mdx

```diff
@@ -20,16 +23,62 @@ Namespaces are defined using the [`DEFINE NAMESPACE`](/docs/surrealdb/surrealql/
 DEFINE NAMESPACE [ OVERWRITE | IF NOT EXISTS ] @name [ COMMENT @string ]
 ```
 
-You can also view the list of namespaces in your SurrealDB instance using the [`INFO`](/docs/surrealdb/surrealql/statements/info) statement. This statement provides information about the accesses, databases, and users present in the current SurrealDB instance.
+You can also view the list of namespaces in your SurrealDB instance using the [`INFO`](/docs/surrealdb/surrealql/statements/info) statement. This statement provides information about the databases, tokens and users present in the current namespace in the SurrealDB instance using the [`NAMESPACE`](/docs/surrealdb/surrealql/statements/info) or [`NS`](/docs/surrealdb/surrealql/statements/info) keyword.
```

Comment:
In 2.0, we have depreciated `DEFINE TOKEN` AND `DEFINE SCOPE` in favour of `DEFINE ACCESS` so you will see accesses as part of the output 
<img width="837" alt="Screenshot 2024-08-29 at 11 36 27" src="https://github.com/user-attachments/assets/d2c586d5-455a-43f1-8d71-9994406a92f4">

```suggestion
You can also view the list of namespaces in your SurrealDB instance using the [`INFO`](/docs/surrealdb/surrealql/statements/info) statement. This statement provides information about the accesses, databases and users present in the current namespace of the SurrealDB instance using the [`NAMESPACE`](/docs/surrealdb/surrealql/statements/info) or [`NS`](/docs/surrealdb/surrealql/statements/info) keyword.
```

Reply:
@Ekwuno so true, im still thinking as if we were in v1. good catch

@Dhghomon @Ekwuno brings up a good point that the demo dataset is still on v1, what do you suggest we do?

Reply:
Sorry I was mistaken it's not the dataset that's V1 its the environment. Which will be updated as we go towards `2.0` GA. I think we can leave the wording as it is for now but remove the dataset link in favour of bash output